src/ScheduledTask/Webhook/OrderClosedWebhookHandler.php

```diff
@@ -0,0 +1,113 @@
+<?php declare(strict_types=1);
+/**
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Payment Module
+ *
+ * Copyright (c) 2022 Adyen N.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ *
+ * Author: Adyen <shopware@adyen.com>
+ */
+
+namespace Adyen\Shopware\ScheduledTask\Webhook;
+
+use Adyen\Shopware\Entity\Notification\NotificationEntity;
+use Adyen\Shopware\Service\AdyenPaymentService;
+use Adyen\Shopware\Service\Repository\AdyenPaymentRepository;
+use Psr\Log\LoggerInterface;
+use Shopware\Core\Checkout\Order\Aggregate\OrderTransaction\OrderTransactionEntity;
+use Shopware\Core\Checkout\Order\Aggregate\OrderTransaction\OrderTransactionStateHandler;
+use Shopware\Core\Framework\Context;
+
+class OrderClosedWebhookHandler implements WebhookHandlerInterface
+{
+    /**
+     * @var AdyenPaymentService
+     */
+    private $adyenPaymentService;
+
+    /**
+     * @var AdyenPaymentRepository
+     */
+    private $adyenPaymentRepository;
```

Comment:
As follow-up to the first comment related to the repository and the service file, could you please double check this usage?